LICENSE.txt

```diff
@@ -1,52 +1,28 @@
-[The "BSD 3-clause license"]
-Copyright (c) 2012-2017 The ANTLR Project. All rights reserved.
+Copyright (c) 2012-2022 The ANTLR Project. All rights reserved.
```

Comment:
Is it possible to update the end year for all files in the repository? (2017 -> 2022)

Reply:
I am sure it is possible to do that although I would not be the right person to do that...  being a non-dev and not having an IDE or other tooling to help update files in bulk.  I know that copyright rules can differ between jurisdictions, but I would think that the correct way to update copyright on each file is as it is worked on.  In other words,  do not claim 2022 copyright on a file last updated in (say) 2019.

I updated the date on the license file because it applies to the project as a whole.  More importantly,  it just seems crazy to be making changes to the license file and and yet neglect to take the opportunity to tweak the date.